flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -468,9 +468,13 @@ public void close() {
     }
 
     @Override
+    @Deprecated
```

Comment:
Should be applied to the implemented interface (`ClusterDescriptor#deployJobCluster`)

Reply:
Done.